DigitalLearningSolutions.Data/Helpers/LinkedFieldHelper.cs

```diff
@@ -7,60 +7,60 @@
 
     public static class LinkedFieldHelper
     {
-        public static List<LinkedFieldNumberWithValues> GetLinkedFieldNumbersWithValuesFromChangedAnswers(
+        public static List<LinkedFieldChange> GetLinkedFieldChanges(
             DelegateUser oldDelegateDetails,

```

Comment:
Yes, so the DelegateUser properties you use here are the same as the properties on CentreAnswersData right?
So you could change this parameter to `CentreAnswersData oldAnswers`, and this method would become easier to use (because you don't need an entire delegate to use it). The signature would also be less surprising to future devs.
Then you can add a method `DelegateUser.GetCentreAnswersData` to extract the answers from your delegate before you call this method (I previously suggested an implicit method, but on second thoughts the idea of casting a `DelegateUser` into a `CentreAnswersData` is a bit strange).